src/webgpu/util/f32_interval.ts

```diff
@@ -335,12 +368,6 @@ function runBinaryOp(x: F32Interval, y: F32Interval, op: BinaryToIntervalOp): F3
     [x, y] = op.extrema(x, y);
   }
 
-  if (op.domain !== undefined) {
```

Comment:
Changing this is why that one unit test is failing. Specifically division has an extrema @ 0, which isn't being handled, but the use of intervals at this point was catching the issue. I will fix the PR to add an extrema? to division

Reply:
Ack

Reply:
Done